torch/distributed/distributed_c10d.py

```diff
@@ -2034,6 +2038,14 @@ def _new_process_group_helper(
                     backend_class.size(),
                 )
                 pg._set_default_backend(backend_type)
+        elif _use_torchcomm: 
```

Comment:
i'm not sure what option we have besides adding a torchcomm path here.  however, we need to align all the kwargs of _new_process_group_helper to support torchcomms equivalents, and it gets messy in certain cases, like backend_options having different formats.  Any thoughts about how to clean this up?